ipv6/miredo/Makefile

```diff
@@ -0,0 +1,59 @@
+# Developer notes:
+#   - Make sure you have gawk, othwerise you get weird syntax errors about "("
+#     and ")".
+#   - Install ccache, otherwise you'll get configure error:
+#     "C compiler cannot create executables"
+#   - Shared libraries are a pain, so we just compile the binary static.
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=miredo
+PKG_VERSION:=1.2.6
+PKG_RELEASE:=1
+
+PKG_SOURCE:=miredo-1.2.6.tar.xz
+PKG_SOURCE_URL:=https://www.remlab.net/files/miredo/
+PKG_HASH:=fa26d2f4a405415833669e2e2e22677b225d8f83600844645d5683535ea43149
+PKG_LICENSE:=GPL-2.0
+PKG_LICENSE_FILES:=COPYING
+PKG_MAINTAINER:=Petr Pudlak <petr.mvd@gmail.com>
+
+PKG_INSTALL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+CONFIGURE_ARGS+=--with-pic --without-libiconv-prefix --without-libintl-prefix --without-Judy
+TARGET_CFLAGS+=-std=gnu99 -O3 -ffast-math
```

Comment:
Why is -O3 and -ffast-math needed here? If it is a performance optimization please add some numbers to the comments or explain why this is good.

Reply:
Removed. I copied it from some other package when creating this one, probably not needed at all.